Screens/PropertyAISettingsScreen.tsx

```diff
@@ -34,6 +39,76 @@ type SettingsSectionProps = {
   style?: any;
 };
 
+const ClearAIChatGroup = () => {
+  const { state } = useUserContext();
+  const [selectedOption, setSelectedOption] = useState<string | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const clearGroup = async () => {
+    if (!selectedOption) return; // Ensure a valid option is selected
+
+    try {
+      setIsLoading(true);
+      const res = await customFetch(Endpoints.clearAIChat, {
+        method: "POST",
+        body: JSON.stringify({
+          groupId: state.currentGroup,
+          smsWebOrBoth: selectedOption,
+        }),
+      });
+      const data = await res.json();
+      if (res.ok) {
+        alert("Group cleared successfully.");
+        setSelectedOption(null); // Reset selection after clearing
+      } else {
+        console.error("Error clearing group: ", data.error);
+      }
+    } catch (error) {
+      console.error("Network error: ", error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  return (
+      <SettingsSection title={"Clear AI Chats"} style={{ flex: 1 }}>
+      <View style={{ flexDirection: "row", justifyContent: "space-around", marginTop: 10}}>
+        <Button
+          text="Web Sessions Only"
+          onPress={() => setSelectedOption("Web")}
+          style={{
+            margin: 5,flex: 1,backgroundColor: selectedOption === "Web" ? colors.purple : colors.gray,
+          }}
+        />
+        <Button
+          text="SMS Numbers Only"
+          onPress={() => setSelectedOption("SMS")}
+          style={{
+            margin: 5,flex: 1, backgroundColor: selectedOption === "SMS" ? colors.purple : colors.gray,
+          }}
+        />
+        <Button
```

Comment:
Can we make both of them selectable? And if both are pressed, then selected option becomes "Both"? Then we won't need the "Both" button